source/agora/network/NetworkManager.d

```diff
@@ -279,9 +279,8 @@ public class NetworkManager
         }
         catch (Exception ex)
         {
-            scope (failure) assert(0);
             log.error("Couldn't retrieve blocks: {}. Will try again later..",
-                ex.message);
```

Comment:
Whoever didn't make it `nothrow` deserves a slap. I think that means either Mihails or Luca.